src/bson.c

```diff
@@ -574,6 +574,8 @@ static const bson_visitor_t php_bson_visitors = {
    php_phongo_bson_visit_int64,
    php_phongo_bson_visit_maxkey,
    php_phongo_bson_visit_minkey,
+   NULL /*php_phongo_bson_visit_unsupported_type*/,
+   NULL /*php_phongo_bson_visit_decimal128*/,
```

Comment:
Because you set this BSON_DECIMAL128 flag above, would it make sense to do a conditional define for it here anyway too?